.github/workflows/ci_test-conda.yml

```diff
@@ -62,11 +62,11 @@ jobs:
         conda list
         pip --version
         python ./requirements/adjust-versions.py requirements.txt
-        python ./requirements/adjust-versions.py requirements/image.txt
+        python ./.github/prune-packages.py requirements/image.txt torchvision
         pip install --requirement requirements.txt --upgrade-strategy only-if-needed --quiet
-        pip install --requirement requirements/test.txt --upgrade-strategy only-if-needed --quiet --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install --requirement requirements/test.txt --upgrade-strategy only-if-needed --quiet
```

Comment:
Just so you know `--upgrade-strategy only-if-needed` is the default, no need to set it 🌵